roles/cdi/README.md

```diff
@@ -0,0 +1,17 @@
+# Ansible role: cdi
+
+This role deploys the CDI controller and if configured will trigger
+an image download and import through CDI.  Below are user controlled 
+parameters.
+
+## Configuration parameters
+* **cdi_image_namespace**: The namespace into which the CDI components should be
+  installed.  Default is `golden`
+* **action**: Currently only supporting action of `deploy`
+
```

Comment:
They have switched to a table notation for the variables documentation.  See [the cns role](https://github.com/kubevirt/kubevirt-ansible/blob/master/roles/storage-cns/README.md) for an example.